rules/style.js

```diff
@@ -0,0 +1,495 @@
+     */
+    camelcase: 0,
+
+    /**
+     * enforce spacing before and after comma
+     * @see http://eslint.org/docs/rules/comma-spacing
+     */
+    'comma-spacing': [2, { before: false, after: true }],
+
+    /**
+     * enforce one true comma style
+     * @see http://eslint.org/docs/rules/comma-style
+     * @since 3.0.0
+     * @example
+     *
+     * ```js
+     * // 错误
+     * var foo = 1
+     * ,
+     * bar = 2;
+     *
+     * var foo = 1
+     * , bar = 2;
+     *
+     * var foo = ["apples"
+     * , "oranges"];
+     *
+     * function bar() {
+     *   return {
+     *     "a": 1
+     *     ,"b:": 2
+     *   };
+     * }
+     * ```
+     */
+    'comma-style': [2, 'last'],
+
+    /**
+     * allow padding inside computed properties
+     * @see http://eslint.org/docs/rules/computed-property-spacing
+     */
+    'computed-property-spacing': 0,
+
+    /**
+     * not enforces consistent naming when capturing the current execution context
+     * @see http://eslint.org/docs/rules/consistent-this
+     */
+    'consistent-this': 0,
+
+    /**
+     * enforce newline at the end of file, with no multiple empty lines
+     * @see http://eslint.org/docs/rules/eol-last
+     */
+    'eol-last': 2,
+
+    /**
+     * not require function expressions to have a name
+     * @see http://eslint.org/docs/rules/func-names
+     */
+    'func-names': 0,
+
+    /**
+     * not enforces use of function declarations or expressions
+     * @see http://eslint.org/docs/rules/func-style
+     */
+    'func-style': 0,
+
+    /**
+     * Blacklist certain identifiers to prevent them being used
+     * @see http://eslint.org/docs/rules/id-blacklist
+     */
+    'id-blacklist': 0,
+
+    /**
+     * not enforces minimum and maximum identifier lengths
+     * (variable names, property names etc.)
+     * @see http://eslint.org/docs/rules/id-length
+     */
+    'id-length': 0,
+
+    /**
+     * not require identifiers to match the provided regular expression
+     * @see http://eslint.org/docs/rules/id-match
+     */
+    'id-match': 0,
+
+    /**
+     * this option sets a specific tab width for your code
+     * @see http://eslint.org/docs/rules/indent
+     */
+    indent: [2, 2, { SwitchCase: 1, VariableDeclarator: 1 }],
+
+    /**
+     * enforces spacing between keys and values in object literal properties
+     * @see http://eslint.org/docs/rules/key-spacing
+     */
+    'key-spacing': [2, { beforeColon: false, afterColon: true }],
+
+    /**
+     * require a space before & after certain keywords
+     * @see http://eslint.org/docs/rules/keyword-spacing
+     */
+    'keyword-spacing': [2, {
+      before: true,
+      after: true,
+      overrides: {
+        return: { after: true },
+        throw: { after: true },
+        case: { after: true },
+      },
+    }],
+
+    /**
+     * allow mixed 'LF' and 'CRLF' as linebreaks
+     * @see http://eslint.org/docs/rules/linebreak-style
+     */
+    'linebreak-style': 0,
```

Comment:
https://help.github.com/articles/dealing-with-line-endings/#global-settings-for-line-endings